src/cmd/build.js

```diff
@@ -214,6 +223,7 @@ export default async function build(
       sourceDir,
       artifactsDir,
       ignoreFiles,
+      filename,
```

Comment:
@birtony it looks that this part of the changes in this PR is still triggering a flow check error: https://travis-ci.org/mozilla/web-ext/jobs/603639960#L242

I haven't looked too deeply into the PR yet, but it looks that Flow is complaining that the type signature of the options passed to `createFileFilter` (which is named `FileFilterOptions`) has not been updated:
https://github.com/mozilla/web-ext/blob/065188db35a4f63c2d17ff744cc1de2601a62222/src/util/file-filter.js#L25-L30

(as a side note, don't worry about the failure logged in the jobs executed by the travis windows workers, it is unrelated to you change and we fixed it as part of #1742, if you are curious #1739 provides some additional details about that issue).

Reply:
@rpl thank you for your suggestions! I have truly included `filename` in the wrong spot, as it seems to be not needed in `createFileFilter`. I have also provided required tests to cover the new feature in a2286ec.